main.go

```diff
@@ -176,8 +167,11 @@ func main() {
 	kingpin.HelpFlag.Short('h')
 	kingpin.Parse()
 	logger := promlog.New(promlogConfig)
+	if *concurrency < 1 {
+		*concurrency = 1
+	}
 
-	level.Info(logger).Log("msg", "Starting snmp_exporter", "version", version.Info())
+	level.Info(logger).Log("msg", "Starting snmp_exporter", "version", version.Info(), "concurrency", concurrency)
```

Comment:
Let's make this a separate log line. We're probably moving this particular log line into the `exporter-toolkit` package.

Reply:
I don't have an image of the implementation, will you be using this one?
https://pkg.go.dev/github.com/prometheus/exporter-toolkit@v0.10.0/web

Reply:
I'm working on a top-level `toolkit` package. It's [still a work in progress](https://github.com/prometheus/exporter-toolkit/pull/147).